build-scripts/build_rule_playbooks.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python3
+
+import argparse
+import ssg.playbook_builder
+
+
+def parse_args():
+    p = argparse.ArgumentParser()
+    p.add_argument(
+        "--input-dir", required=True, dest="input_dir",
+        help="Input directory that contains all Ansible remediations"
+        "snippets for the product we are building. "
+        "e.g. ~/scap-security-guide/build/fedora/fixes/ansible"
+    )
+    p.add_argument(
+        "--output-dir", required=True, dest="output_dir",
+        help="Output directory to which the rule-based Ansible playbooks "
+        "will be generated. "
+        "e.g. ~/scap-security-guide/build/fedora/playbooks"
+    )
```

Comment:
Ditto, we know a sensible default for the output directory in a similar way as with the input dir.

Reply:
OK, I'm not sure if we will need them when the script is used standalone, so I created an issue as you suggested. See #4034